src/checkPassword.test.js

```diff
@@ -1,19 +1,38 @@
-'use strict';
+"use strict";
 
 describe(`Function 'checkPassword':`, () => {
-  const checkPassword = require('./checkPassword');
+  const checkPassword = require("./checkPassword");
 
   it(`should be declared`, () => {
     expect(checkPassword).toBeInstanceOf(Function);
   });
 
   it(`should return boolean`, () => {
-
+    expect(checkPassword("4134")).toBe(false);
   });
 
   it(`should return 'true' for the valid password with 8 characters`, () => {
+    expect(checkPassword("Password1!")).toBe(true);
+  });
 
+  it(`should return 'true' for the valid password with 16 characters`, () => {
+    expect(checkPassword("Password1!123456")).toBe(true);
   });
 
-  // write more tests here
+  it(`should return 'true' for the valid password with
+    contains at least 1 digit, 1 special character, 1 uppercase letter`, () => {
```

Comment:
The test description is incomplete and should be rephrased for clarity. Consider changing it to: `should return 'true' for a valid password that contains at least 1 digit, 1 special character, and 1 uppercase letter`.